src/syscheckd/db/include/dbFileItem.hpp

```diff
@@ -14,36 +14,11 @@
 #include "json.hpp"
 #include "dbItem.hpp"
 
-struct FimFileDataDeleter
```

Comment:
Why do you delete this ?

Reply:
These part of the code has been removed because we need the information retrieved by `createFimEntry` in C. Using the smart deleters, there was a problem when the dbItem object is out of the scope, this memory is deallocated and it's no longer valid. That is the reason why the smart deleters are removed

Reply:
This change greatly affects what I have done because I use c ++ smart pointers. I would change to back version until my pull request will be merged so I will review again it after that

Reply:
It is recommended, manage the memory part from c ++